R/reportCrossVariables.R

```diff
@@ -63,24 +60,11 @@ reportCrossVariables <- function(gdx, output = NULL, regionSubsetList = NULL,
   budget.m <- budget.m[,y,]
   sebal.m  <- sebal.m[,y,]
   pm_ts    <- pm_ts[,y,]
-  ####### internal function for cumulated values ############
-  cumulatedValue <- function(var,i_pm_ts=pm_ts){
```

Comment:
this internal function was not used in this function?

Reply:
no, it wasn't